packages/google_maps_flutter/google_maps_flutter_platform_interface/test/types/ground_overlay_test.dart

```diff
@@ -0,0 +1,403 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:google_maps_flutter_platform_interface/google_maps_flutter_platform_interface.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  group('$GroundOverlay', () {
```

Comment:
@stuartmorgan IIRC this should just be 'GroundOverlay' because the `$` messes up test parsing in other IDEs?

Reply:
Ah, yes, thanks for catching that.

Someday I should sweep all of our existing tests to remove this pattern so it doesn't get replicated forever.